eclipsePlugin/about.properties

```diff
@@ -1,4 +1,6 @@
 blurb=SpotBugs \n\
 \n\
-(c) SpotBugs Project. Licensed under the LGPL.\n\
-https://spotbugs.github.io\n\
+Copyright (c) 2017 SpotBugs Project. All rights reserved.\n\
+Licensed under the LGPL.\n\
+Copyright 2003-2008 by the University of Maryland. \n\
```

Comment:
Regarding the date: I just kept what was in the file before. I'm not a layer, so just trying to be polite.
Regarding the lines: OK.